chips/imxrt1050/src/deferred_call_tasks.rs

```diff
@@ -0,0 +1,30 @@
+//! Definition of Deferred Call tasks.
+//!
+//! Deferred calls allow peripheral drivers to register pseudo interrupts.
+//! These are the definitions of which deferred calls this chip needs.
+
+use core::convert::Into;
+use core::convert::TryFrom;
+
+/// A type of task to defer a call for
+#[derive(Copy, Clone)]
+pub enum Task {
+    Nop = 0,
+}
+
+impl TryFrom<usize> for Task {
+    type Error = ();
+
+    fn try_from(value: usize) -> Result<Task, ()> {
+        match value {
+            0 => Ok(Task::Nop),
+            _ => Err(()),
+        }
+    }
+}
+
+impl Into<usize> for Task {
+    fn into(self) -> usize {
+        self as usize
+    }
+}
```

Comment:
If there are no deferred calls in use for this chip yet, I am not sure we should include this file or the other machinery in `chip.rs` for deferred call tasks (unless you are using deferred calls out of tree and plan to add those in subsequent PRs).

Reply:
I do not use them, so I deleted it.